File: Programming Basics/Programming Basics - C#/Exams/Coding 101 Exam - 26 March 2016/Exam - 26 March 2016/3. Trip/Trip.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace _3.Trip
{
    class Trip
    {
        static void Main(string[] args)
        {
            double monney = double.Parse(Console.ReadLine());
            var season = Console.ReadLine();

            double percentage = 0;
            if (monney <= 100)
            {
                if (season == "summer")
                {
                    percentage = monney * 0.30;
                    Console.WriteLine("Somewhere in Bulgaria");
                    Console.WriteLine("Camp - " + string.Format("{0:f2}", percentage));
                }

                else
                {
                    percentage = monney * 0.70;
                    Console.WriteLine("Somewhere in Bulgaria");
                    Console.WriteLine("Hotel - " + string.Format("{0:f2}", percentage));
                }
            }

            else if (monney > 100 && monney <= 1000)
            {
                if (season == "summer")
                {
                    percentage = monney * 0.40;
                    Console.WriteLine("Somewhere in Balkans");
                    Console.WriteLine("Camp - " + string.Format("{0:f2}", percentage));
                }

                else
                {
                    percentage = monney * 0.80;
                    Console.WriteLine("Somewhere in Balkans");
                    Console.WriteLine("Hotel - " + string.Format("{0:f2}", percentage));
                }
            }

            else
            {
                percentage = monney * 0.90;
                Console.WriteLine("Somewhere in Europe");
                Console.WriteLine("Hotel - " + string.Format("{0:f2}", percentage));
            }
        }
    }
}

File: Programming Basics/Programming Basics - JavaScript/Exams/Coding 101 Exam - 26 March 2016/3. Trip/Trip.js
function moneyForTrip([arg1, arg2]) {
    let budget = Number(arg1);
    let season = arg2;

    let price;
    let location;
    let camapHotel;
    if (budget <= 100) {
        if (season == "summer") {
            price = budget * 0.30;
            location = "Bulgaria";
            camapHotel = "Camp";
        }
        else {
            price = budget * 0.70;
            location = "Bulgaria";
            camapHotel = "Hotel";
        }
    }

    else if (budget > 100 && budget <= 1000) {
        if (season == "summer") {
            price = budget * 0.40;
            location = "Balkans";
            camapHotel = "Camp";
        }
        else {
            price = budget * 0.80;
            location = "Balkans";
            camapHotel = "Hotel";
        }
    }

    else {
        price = budget * 0.90;
        location = "Europe";
        camapHotel = "Hotel";
    }

    console.log(`Somewhere in ${location}`);
    console.log(`${camapHotel} - ${price.toFixed(2)}`);
}

/* Uncomment to test:
moneyForTrip(["50", "summer"]);
moneyForTrip(["75", "winter"]);
moneyForTrip(["312", "summer"]);
moneyForTrip(["678.53", "winter"]);
moneyForTrip(["1500", "summer"]);
    */